app/views/projects/index.html.erb

```diff
@@ -0,0 +1,41 @@
+<h1>Projects#index</h1>
+<p>Find me in app/views/projects/index.html.erb</p>
+<div class='container'>
+  <div class='row'>
+    <div class='col'>
+      <table class='table mb-5'>
+        <%= link_to 'Create Project', new_user_project_path, class:'btn btn-success mb-2 float-right text-light' %>
+        <thead class='bg-dark text-white'>
+          <tr>
+            <th scope='col'>Title</th>
+            <th scope='col'>Description</th>
+            <th scope='col'>Active</th>
+            <th scope='col'>Start Date</th>
+          </tr>
+        </thead>
+        <% @projects.each do |project| %>
```

Comment:
it will create `<tbody>` for each project ?

check in the chrome console and fix it.